src/typeWriter.ts

```diff
@@ -16,7 +16,7 @@ export default class TypeWritter {
 
   public restartTypeWriter() {
     this.memoWord = this.nextWord
-    this.eventQueue = this.nextWord.split('')
+    this.eventQueue = this.nextWord
```

Comment:
`eventQueue` should be an Array type because we need to call `Array.shift` to get each character.

What is needed here is to make sure `this.nextWord` has `split` functions on its type, which can be modified to `this.eventQueue = this.nextWord ? this.nextWord.split('') : []` .

Thanks for submitting this PR anyway😉

Reply:
I've pushed the changes you requested, which should fix the functionality if I understood correctly. Please review once again!

Thanks for the prompt reply to this PR! Definitely appreciated  =)